src/Mauro/Greeter.test.ts

```diff
@@ -1,42 +1,15 @@
 import { Greeter } from "./Greeter";
 
 describe("Greeter.greet", () => {
-  it("should receive a name return a message Hello 'name'", () => {
+  it("should return 'Good morning <name>' when the time is 06:00-12:00", () => {
```

Comment:
What do you think about having a test for when the time is out of 6:00 - 12:00 time span?